src/containers/MarkersList.js

```diff
@@ -2,15 +2,16 @@ import React from 'react';
 import MapMarker from '../components/MapMarker';
 
 export default function MarkersList(props) {
+
   const markersOnMap = Object.entries(props.markers).map(([key, marker]) =>
```

Comment:
```suggestion
  const markersOnMap = Object.entries(props.markers).map(([key, { value }]) =>
```
would allow you to get rid of all those ugly `marker.value`s, at the expense / gain of using destructing syntax.